plugins/modules/ipasudorule.py

```diff
@@ -556,6 +580,9 @@ def main():
                     if hostgroup is not None:
                         hostgroup_add = gen_add_list(
                             hostgroup, res_find.get("memberhost_hostgroup"))
+                    if hostmask is not None:
```

Comment:
hostmask can not be None due to line 372.

Reply:
fixed.